client/components/data/query-contact-details-cache/index.jsx

```diff
@@ -16,9 +16,16 @@ import { requestContactDetailsCache } from 'state/domains/management/actions';
 
 class QueryContactDetailsCache extends Component {
 	componentWillMount() {
-		if ( this.props.isRequesting || ! isEmpty( this.props.contactDetailsCache ) ) {
+		const {
+			isRequesting,
+			contactDetailsCacheIsEmpty,
+		} = this.props;
+
+		if ( isRequesting || ! contactDetailsCacheIsEmpty ) {
 			return;
 		}
+
+		// Not destructured to disambiguate unbound action
```

Comment:
I feel like, if we start adding these types of comments, we'll end up with a big pile of unnecessary justifications. If someone actually sees this and cares so much that they'll try and "fix it", ESLint will yell at them, so it's fine. :) Let's remove the comment.